libr/bin/bfile.c

```diff
@@ -794,34 +799,33 @@ R_IPI RList *r_bin_file_get_strings(RBinFile *a, int min, int dump, int raw) {
 				if (section->size > a->size) {
 					continue;
 				}
+				ut8 *sbuf = malloc (section->size);
+				if (!sbuf) {
+					continue;
+				}
+				r_buf_read_at (a->buf, section->paddr + cfstr_offs, sbuf, section->size);
```

Comment:
why reading the entire section instead of doing what was done before?

Reply:
its faster

Reply:
yes but it could also be a loooot of data (think about a data section with many many resources inside..)